NEWgIT.Client/Pages/Index.razor

```diff
@@ -0,0 +1,164 @@
+@using System.Net.Http;
+@using NEWgIT.Core.Data;
+@using System.Text.Json;
+@using System.Text.Json.Serialization;
+@using System.Web.Mvc;
+
+
+<RadzenText TextStyle="TextStyle.H2" TagName="TagName.H1" Class="my-4" TextAlign="TextAlign.Center">
+    Welcome to NEWgIT
+</RadzenText>
+
+<RadzenText TextStyle="TextStyle.Subtitle2" TagName="TagName.H3" TextAlign="TextAlign.Center"></RadzenText>
+<RadzenTextBox Placeholder="Insert github repository identifier" Change=@(args => OnChange(args)) Class="TextBox" />
+
+ <RadzenButton class="AnalyzeButton" Click=@(args => OnClick("Primary button")) Text="Analyze" ButtonStyle="ButtonStyle.Primary"/>
+@if(value!=""){
+<RadzenText TextStyle="TextStyle.H4" TagName="TagName.H1" Class="my-4" TextAlign="TextAlign.Center">
+    Commit Frequency
+</RadzenText>
+
+<RadzenChart>
+    <RadzenColumnSeries Data="@allCommits" CategoryProperty="Date" Title="All Commits" LineType="LineType.Dashed" ValueProperty="commits">
+    </RadzenColumnSeries>
+    <RadzenColumnOptions Radius="5" />
+    <RadzenValueAxis>
+        <RadzenGridLines Visible="true" />
+    </RadzenValueAxis>
+</RadzenChart> 
+
+<RadzenText TextStyle="TextStyle.H4" TagName="TagName.H1" Class="my-4" TextAlign="TextAlign.Center">
+    Commit Frequency pr author
+</RadzenText> 
+
+@foreach(var author in commitsPrAuthor)
+{ 
+<RadzenChart>
+    <RadzenColumnSeries Data="@author.Value" CategoryProperty="Date" Title="@author.Key" LineType="LineType.Dashed" ValueProperty="commits">
+    </RadzenColumnSeries>
+    <RadzenColumnOptions Radius="5" />
+    <RadzenValueAxis>
+        <RadzenGridLines Visible="true" />
+    </RadzenValueAxis>
+</RadzenChart> 
+} 
+
+ <RadzenDataGrid AllowColumnResize="true" PageSize="5" ShowPagingSummary="true" Data="@Forks" TItem="string" ColumnWidth="300px" LogicalFilterOperator="LogicalFilterOperator.Or">
+        <Columns>
+            <RadzenDataGridColumn TItem="String" Property="String" Filterable="false" Title="Forks" Frozen="true" Width="80px" TextAlign="TextAlign.Center" />
+        </Columns>
+</RadzenDataGrid>
+}
+
+
+@code {
+//MOCK
+ICollection<string>? Forks;
+Dictionary<DateOnly, int>? frequencyDictionary;
+Dictionary<string, Dictionary<DateOnly, int>>? authorDictionary;
+Dictionary<string, commitItem[]> commitsPrAuthor = new Dictionary<string, commitItem[]>();
+string[] commitsPrAuthors;
+string? value = "";
+string? repoIdentifier; 
+commitItem[] allCommits;
+int stupidFixer = 0; 
+
+void OnChange(string value)
+{
+    this.value = value;
+}
+static readonly HttpClient client = new HttpClient();
+
+async void OnClick(string buttonName)
+{   
+    analyseAllCommits();
+    analysePrAuthor();
+    analyseForks();
+} 
+
+async void analyseAllCommits(){
+    try
+    {
+        HttpResponseMessage response = await client.GetAsync($"https://localhost:7235/analysis/{value}/frequency");
+        response.EnsureSuccessStatusCode();
+
+        var responseBody = await response.Content.ReadFromJsonAsync<FrequenciesDTO>();

```

Comment:
Bruger vi de samme DTO klasser her som backenden sender ud? Det skal vi lige have diskuteret. De to sider burde gerne kunne fungere helt separat